src/storm-dft-cli/storm-dft.cpp

```diff
@@ -114,6 +115,66 @@ void processOptions() {
     }
 #endif
 
+    // BDD Analysis
+    if(dftIOSettings.isExportToBddDot() ||
+            dftIOSettings.isAnalyzeWithBdds() ||
+            dftIOSettings.isMinimumCutSets() ||
+            dftIOSettings.isImportanceMeasureSet()) {
+        bool const isImportanceMeasureSet{dftIOSettings.isImportanceMeasureSet()};
+        bool const isMinimumCutSets{dftIOSettings.isMinimumCutSets()};
+        bool const isMTTF{dftIOSettings.usePropExpectedTime()};
+        bool const isExportToBddDot{dftIOSettings.isExportToBddDot()};
+        bool const isTimebound{dftIOSettings.usePropTimebound()};
+        bool const isTimepoints{dftIOSettings.usePropTimepoints()};
+
+        bool const probabilityAnalysis {ioSettings.isPropertySet() || !isImportanceMeasureSet};
```

Comment:
I don't see the problem. How do you trigger the segfault?

`storm-dft --bdd` does not seem to do the trick

Reply:
`storm-dft --bdd --modularisation` (and a DFT file of course) triggers the segfault for me. The problem comes from [api/storm-dft.cpp](https://github.com/moves-rwth/storm/blob/56b1b52ee91eddccff88591cf794376c95205235/src/storm-dft/api/storm-dft.cpp#L37).

Reply:
That segfault was the uninited workDFT in `DFTModularizer::check`.
Funny is that it is only called because the call was not guarded from an empty formula vector, which you commented on.
All check functions shall behave if called with an empty request (like an empty property vector or timepoints vector).
Therefore I see no violation in the call to analyzeDFTBdd.

Reply:
If they work with empty requests, then we can leave it as is.